website/docs/concepts/canbus_service/README.md

```diff
@@ -40,6 +44,10 @@ motors accordingly.
 3. `/get_battery_state`: Returns the battery state.
 A float value indicating the battery's state of charge, ranging [0.0, 1.0].
 If no motors are connected, it returns -1.0.
+4. `/control_tools`: Control the tools / actuators based on the ActuatorCommands message.
+5. `stop_all_tools`: Stop all tools and prevent control for a predetermined duration.
+6. `yield_all_tools`: Prevent control for a predetermined duration.
+7. `config_request`: Use the farm-ng SDO protocol to send a config request to the dashboard.
```

Comment:
These should also have `/` right?
```suggestion
5. `/stop_all_tools`: Stop all tools and prevent control for a predetermined duration.
6. `/yield_all_tools`: Prevent control for a predetermined duration.
7. `/config_request`: Use the farm-ng SDO protocol to send a config request to the dashboard.
```

Reply:
Good catch! Fixed here 70cae79b0c6dc1411e1fc0fcc13dc8f22811c324